.travis.yml

```diff
@@ -12,4 +12,5 @@ rvm:
   - 2.3
   - 2.4
   - 2.5
-
+  - 2.6
```

Comment:
might have to use 2.6.1 or 2.6.0 ... not sure why it does not work ... or google about supported travis ruby versions :/

Reply:
... maybe remove the dist+sudo parts ...

Reply:
Thanks quick response. It may be that precise dist is too old.

Reply:
I see. it should consider http://rubies.travis-ci.org/